Symfony/CS/Tests/Fixer/Symfony/ArrayElementNoSpaceBeforeCommaFixerTest.php

```diff
@@ -78,26 +78,26 @@ public function testFixProvider()
             // multi line array
             array(
                 '<?php $x = [  "a" =>$a,
-                    "b"=>  
+                    "b"=>
```

Comment:
Are you sure having the trailing white space is not by design? (i.e. it tesst it is supported by the fixer being tested?)